db/migrate/20150812105455_add_default_to_content_field.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToContentField < ActiveRecord::Migration
+  def change
+    change_column :campaign_pages, :content, :text, default: ''
+  end
```

Comment:
Newly created campaign pages had nil for content, which was a hassle to deal with. Probably best you delete existing pages after this migration.